nrf_wifi/fw_if/umac_if/inc/fw/patch_info.h

```diff
@@ -57,6 +57,6 @@ struct nrf70_fw_image_info {
 #define RPU_FAMILY         (1)
 #define RPU_MAJOR_VERSION   (2)
 #define RPU_MINOR_VERSION   (9)
-#define RPU_PATCH_VERSION   (9)
+#define RPU_PATCH_VERSION   (20)
```

Comment:
Woah, a big jump, any specific reason?

Reply:
Reserved 10 to 19 for bug fix releases as per below comment.
https://nordicsemi.atlassian.net/browse/SHEL-2220?focusedCommentId=1725578

Reply:
Thanks, can you please add a comment explaining that 10-19 versions are reserved.

Reply:
@udaynordic , We have already provided version 1.2.9.10 to NCS2.5.1 release and for further fixes there we reserved 10-19.

Reply:
oh got it.

Reply:
updated

Reply:
I don't see a comment?